command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
To clarify what I said (and agree with what @wking said in parallel :)) we decided to specify a cli because tools like compliance testers and orchestrators need it. That doesn't mean - to @duglin's concern - that you MUST implement the cli API to be compliant: you may choose to just accept runtime spec bundles and run them via another API. But IF you want to be used by an orchestrator or compliance testing tool in a pluggable way then you MUST implement the cli spec. We may later in theory have other specs like a web spec but tbh I agree we can blow up that bridge when we come to it and prefer just specifying the cli and saying (which is a statement of necessary fact today) that if you want to be certified you need to implement this spec so the certification tool can invoke you.